corehq/apps/geospatial/models.py

```diff
@@ -135,13 +136,40 @@ class GeoConfig(models.Model):
 
     CUSTOM_USER_PROPERTY = 'custom_user_property'
     ASSIGNED_LOCATION = 'assigned_location'
+    RADIAL_ALGORITHM = 'radial_algorithm'
+    ROAD_NETWORK_ALGORITHM = 'road_network_algorithm'
+    MIN_MAX_GROUPING = 'min_max_grouping'
+    TARGET_SIZE_GROUPING = 'target_size_grouping'
 
     VALID_LOCATION_SOURCES = [
         CUSTOM_USER_PROPERTY,
         ASSIGNED_LOCATION,
     ]
+    VALID_DISBURSEMENT_ALGORITHMS = [
+        (RADIAL_ALGORITHM, _('Radial Algorithm')),
+        (ROAD_NETWORK_ALGORITHM, _('Road Network Algorithm')),
+    ]
+    VALID_GROUPING_METHODS = [
+        (MIN_MAX_GROUPING, _('Min/Max Grouping')),
+        (TARGET_SIZE_GROUPING, _('Target Size Grouping')),
+    ]
 
     domain = models.CharField(max_length=256, db_index=True, primary_key=True)
     location_data_source = models.CharField(max_length=126, default=CUSTOM_USER_PROPERTY)
     user_location_property_name = models.CharField(max_length=256, default=GPS_POINT_CASE_PROPERTY)
     case_location_property_name = models.CharField(max_length=256, default=GPS_POINT_CASE_PROPERTY)
+
+    selected_grouping_method = models.CharField(
```

Comment:
@zandre-eng 
Might be good to add a link to some documentation around what exactly the grouping methods mean. If not docs exist yet, maybe just add a todo in the code or a ticket in the backlog.

Reply:
That's a good idea. There's no documentation around this, so I'll add in a TODO for when this feature gets GA'ed and create a relevant ticket to update the FF documentation.

Reply:
Reminder to please consider updating the README in the geospatial folder where applicable, to give useful information about the features in the geospatial django app.

Reply:
[TODO added](https://github.com/dimagi/commcare-hq/pull/33507/commits/70973373f4dce67de96334d1972258df560d8cf7) and [ticket created](https://dimagi-dev.atlassian.net/browse/SC-3139).